test/e2e/tsconfig.json

```diff
@@ -2,7 +2,7 @@
 	"extends": "@automattic/calypso-build/typescript/ts-package.json",
 	"compilerOptions": {
 		"noEmit": true, // just type checking, no output. The output is handled by babel.
-        "types": [ "jest" ] // no mocha - we are only using TypeScript for the new Playwright scripts
+        "types": [ "jest", "node" ] // no mocha - we are only using TypeScript for the new Playwright scripts
```

Comment:
I don't know enough about TypeScript configuration side of things - what does this do?

Reply:
Type definitions for the node: https://www.npmjs.com/package/@types/node. This was needed for the `process` global. I will remove that line, so I can remove this too.

Reply:
Gotcha. Thanks for clarifying!